code/modules/reagents/reagents/toxins.dm

```diff
@@ -353,7 +353,7 @@
 /datum/reagent/toxin/plantbgone
 	name = "Plant-B-Gone"
 	id = "plantbgone"
-	description = "A harmful toxic mixture to kill plant life."
+	description = "Polytrinic acid is an extremely corrosive chemical substance. Can be used to destroy objects and Hivemind wires."
```

Comment:
This one is just flat wrong it does not clean out hivemind wires as it has direct code to prevent that interaction